pwnlib/heap/arena/arena.py

```diff
@@ -0,0 +1,112 @@
+from pwnlib.heap.bins import NoTcacheError
+from pwnlib.heap.malloc_state import \
+    SMALL_BINS_START_INDEX, \
+    LARGE_BINS_START_INDEX
+from pwnlib.heap.basic_formatter import BasicFormatter
+
+
+class Arena(object):
+    """Class with the information of the arena.
+
+    Attributes:
+        malloc_state (MallocState): The malloc_state struct of the arena
```

Comment:
There is a better way to document attributes, which Sphinx supports:

```py
#: The heap of the arena
self.heap = heap
```

The magic `#:` causes Sphinx to pick up the attribute documentation.  See `elf.py` for more examples.